### PR TITLE
Publish definitions

### PR DESCRIPTION
**Pull request by anxolin**
* Generate typescrypt definitions so it can be used by third parties with typescrypt too
* Point to the declaration in package.json

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gnosis.pm/dex-js",
-  "version": "0.0.2",
+  "version": "0.0.6",
```
Comment (alfetopito) [inline package.json]: Since this is all going in a single PR, can't the version be bumped from 0.0.2 to 0.0.3 rather than 0.0.6?

Comment (anxolin) [reply package.json]: The issue is that for testing the NPM packages, I published different versions already. 0.0.6 is the current one (see the commits)

Comment (anxolin) [reply package.json]: Normally we won't do this after is merged.

- short term: we'll do a version in master branch (develop for dex-react or dex-telegram), and publish that
- later: we'll integrate with travis, so it's travis the one publishing on every new tag

**Review by alfetopito — APPROVED** [review]

File: tsconfig.json [hunk 2]
```diff
@@ -10,6 +10,7 @@
     "moduleResolution": "node",
     "allowJs": false,
     "checkJs": false,
+    "declaration": true,
```
Comment (Velenir) [inline tsconfig.json:13]: add `"declarationMap": true,`

**Review by Velenir — APPROVED** [review]

File: tsconfig.json (hunk 2 above)
Comment (anxolin) [reply tsconfig.json:13]: done 

image

**Pull request merged**